pkg/cmd/cluster/view.go

```diff
@@ -0,0 +1,51 @@
+package cluster
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/buildkite/cli/v3/internal/cluster"
+	"github.com/buildkite/cli/v3/internal/io"
+	"github.com/buildkite/cli/v3/pkg/cmd/factory"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdClusterView(f *factory.Factory) *cobra.Command {
+	var clusterID string
+	cmd := cobra.Command{
+		DisableFlagsInUseLine: true,
+		Use:                   "view  [flags]",
+		Args:                  cobra.MinimumNArgs(0),
```

Comment:
Should this accept an arg instead of a flag for the cluster to view? I think that lines up more with other `view` commands. ie `bk build view <id>` so we should keep it consistent and convert the flag to an argument

Reply:
Updated now